src/main/scala/scalaz/netty/Client.scala

```diff
@@ -75,7 +76,7 @@ private[netty] final class Client(limit: Int) {
 
     override def channelRead(ctx: ChannelHandlerContext, msg: AnyRef): Unit = {
       val buf = msg.asInstanceOf[ByteBuf]
-      val bv = ByteVector(buf.nioBuffer)       // copy data (alternatives are insanely clunky)
+      val bv = ByteVector.view(buf.nioBuffer)       // copy data (alternatives are insanely clunky)
       buf.release()
```

Comment:
The problem with using `view` here is that we're releasing `buf` on the very next line!  This is why I didn't write it this way in the first place.  :-)  The only way to really do a `view` here is if we handle the transcoding from `ByteVector` to the final message type (transcoding based on a `Codec` perhaps) right *here*, releasing the `ByteBuf` only after we're done.

This immediate release, btw, is the reason why the tests are failing on this branch.